packages/storybook/stories/Overview.stories.mdx

```diff
@@ -64,28 +64,128 @@ npm i --save @azure/communication-react
 Composites are higher-level components composed of UI components that deliver turn-key solutions for common communication scenarios using Azure Communication Services.
 Developers can easily instantiate the Composite using an Azure Communication Services access token and the required configuration attributed for call or chat.
 
-| Composite                                                                   | Use Cases                                                                                                                                                                                                                                                                                                    |
-| --------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| [CallComposite](./?path=/story/composites-call-basicexample--basic-example) | Calling experience that allows users to start or join a call. Inside the experience users can configure their devices, participate in the call with video and see other participants, including those with video turn on. For Teams Interop is includes lobby functionality for user to wait to be admitted. |
-| [ChatComposite](./?path=/story/composites-chat-basicexample--basic-example) | Chat experience where user can send and receive messages. Thread events like typing, reads, participants entering and leaving are displayed to the user as part of the chat thread.                                                                                                                          |
+### Call composite
+
+[CallComposite](./?path=/story/composites-call-basicexample--basic-example) is providing a calling experience that allows users to start or join a call. Inside the experience users can configure their
+devices, participate in the call with video and see other participants, including those with video turn on.
+
+Here are visual examples of Call composite where participants have their camera turned off, then some ahve them turned on.
+People speaking with microphone unmuted will have their tile appearing with a blue border.
+
+<Stack
+  horizontal
+  horizontalAlign="space-between"
+  tokens={{ childrenGap: '2rem' }}
+  style={{ backgroundColor: '#EFF6FC', padding: '2.25rem' }}
```

Comment:
pull out this background style color code as a constant somewhere as it's used in multiple places and will want to always be consistent